packages/api/tests/endpoints/granules/test-bulk-reingest.js

```diff
@@ -0,0 +1,288 @@
+const request = require('supertest');
+const sinon = require('sinon');
+const test = require('ava');
+
+const { s3 } = require('@cumulus/aws-client/services');
+const {
+  recursivelyDeleteS3Bucket,
+} = require('@cumulus/aws-client/S3');
+const { randomString } = require('@cumulus/common/test-utils');
+const { EcsStartTaskError } = require('@cumulus/errors');
+
+const { createFakeJwtAuthToken, setAuthorizedOAuthUsers } = require('../../../lib/testUtils');
+
+const models = require('../../../models');
+
+const { app } = require('../../../app');
+
+process.env = {
+  ...process.env,
+  AccessTokensTable: randomString(),
+  CollectionsTable: randomString(),
+  GranulesTable: randomString(),
+  TOKEN_SECRET: randomString(),
+  stackName: randomString(),
+  system_bucket: randomString(),
+  AsyncOperationsTable: randomString(),
+  AsyncOperationTaskDefinition: randomString(),
+  EcsCluster: randomString(),
+  BulkOperationLambda: randomString(),
+  invoke: randomString(),
+  ES_HOST: randomString(),
+  METRICS_ES_HOST: randomString(),
+  METRICS_ES_USER: randomString(),
+  METRICS_ES_PASS: randomString(),
```

Comment:
I prefer randomId, but I know this was a lot of values.  (and they're done that way in the other files)